### ISSUE TITLE
workflow:ERROR [Node inu_n4.a0 failed to run]

### ISSUE THREAD

**Issue by noelkuo** [poldracklab/mriqc#600]
Dear all:
We tried to use mriqc four months ago, and we had a problem (#423). We bought new computer this month, and reinstall mriqc to try it. The data is from BIDS database, unfortunately, we still had similar error. In addition, these are log files In the attachment. Maybe we miss some step, could you help us to solve these questions?  We are grateful for your assistance.

[root@ymu040-233 MRIQC_TEST]# mriqc bids-root/ Output participant
2017-07-25 21:26:26,157 mriqc.cli:WARNING Anonymized quality metrics will be submitted to MRIQC's metrics repository. Use --no-sub to disable submission.
2017-07-25 21:26:27,237 workflow:ERROR ['Node inu_n4.a0 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:26:27,257 workflow:ERROR ['Node inu_n4.a1 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:26:27,761 workflow:ERROR ['Node inu_n4.a1 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:26:27,812 workflow:ERROR ['Node inu_n4.a0 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:31:20,659 workflow:ERROR ['Node SharpenEPI.a1 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:31:23,586 workflow:ERROR ['Node SharpenEPI.a0 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:31:23,595 workflow:ERROR ['Node smoothness.a1 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:31:26,695 workflow:ERROR ['Node smoothness.a0 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:31:28,239 workflow:ERROR ['Node SharpenEPI.a5 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:31:31,317 workflow:ERROR ['Node smoothness.a5 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:31:42,289 workflow:ERROR ['Node SharpenEPI.a2 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:31:45,042 workflow:ERROR ['Node smoothness.a2 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:31:46,434 workflow:ERROR ['Node SharpenEPI.a3 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:31:49,392 workflow:ERROR ['Node smoothness.a3 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:31:52,398 workflow:ERROR ['Node SharpenEPI.a4 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:31:55,189 workflow:ERROR ['Node smoothness.a4 failed to run on host ymu040-233.ym.edu.tw.']
2017-07-25 21:32:05,169 workflow:ERROR could not run node: workflow_enumerator.anatMRIQCT2w.AFNISkullStripWorkflow.inu_n4.a0
2017-07-25 21:32:05,169 workflow:ERROR could not run node: workflow_enumerator.anatMRIQCT2w.AFNISkullStripWorkflow.inu_n4.a1
2017-07-25 21:32:05,169 workflow:ERROR could not run node: workflow_enumerator.anatMRIQCT1w.AFNISkullStripWorkflow.inu_n4.a1
2017-07-25 21:32:05,169 workflow:ERROR could not run node: workflow_enumerator.anatMRIQCT1w.AFNISkullStripWorkflow.inu_n4.a0
2017-07-25 21:32:05,169 workflow:ERROR could not run node: workflow_enumerator.funcMRIQC.SpatialNormalization.SharpenEPI.a1
2017-07-25 21:32:05,170 workflow:ERROR could not run node: workflow_enumerator.funcMRIQC.SpatialNormalization.SharpenEPI.a0
2017-07-25 21:32:05,170 workflow:ERROR could not run node: workflow_enumerator.funcMRIQC.ComputeIQMs.smoothness.a1
2017-07-25 21:32:05,170 workflow:ERROR could not run node: workflow_enumerator.funcMRIQC.ComputeIQMs.smoothness.a0
2017-07-25 21:32:05,170 workflow:ERROR could not run node: workflow_enumerator.funcMRIQC.SpatialNormalization.SharpenEPI.a5
2017-07-25 21:32:05,170 workflow:ERROR could not run node: workflow_enumerator.funcMRIQC.ComputeIQMs.smoothness.a5
2017-07-25 21:32:05,170 workflow:ERROR could not run node: workflow_enumerator.funcMRIQC.SpatialNormalization.SharpenEPI.a2
2017-07-25 21:32:05,170 workflow:ERROR could not run node: workflow_enumerator.funcMRIQC.ComputeIQMs.smoothness.a2
2017-07-25 21:32:05,170 workflow:ERROR could not run node: workflow_enumerator.funcMRIQC.SpatialNormalization.SharpenEPI.a3
2017-07-25 21:32:05,170 workflow:ERROR could not run node: workflow_enumerator.funcMRIQC.ComputeIQMs.smoothness.a3
2017-07-25 21:32:05,170 workflow:ERROR could not run node: workflow_enumerator.funcMRIQC.SpatialNormalization.SharpenEPI.a4
2017-07-25 21:32:05,170 workflow:ERROR could not run node: workflow_enumerator.funcMRIQC.ComputeIQMs.smoothness.a4
Traceback (most recent call last):
  File "/usr/local/bin/mriqc", line 11, in <module>
    load_entry_point('mriqc==0.9.7rc6+7.gdbdb5cc', 'console_scripts', 'mriqc')()
  File "/usr/local/lib/python3.6/site-packages/mriqc/bin/mriqc_run.py", line 321, in main
    workflow.run(**plugin_settings)
  File "/usr/local/lib/python3.6/site-packages/niworkflows/nipype/pipeline/engine/workflows.py", line 590, in run
    runner.run(execgraph, updatehash=updatehash, config=self.config)
  File "/usr/local/lib/python3.6/site-packages/niworkflows/nipype/pipeline/plugins/base.py", line 279, in run
    report_nodes_not_run(notrun)
  File "/usr/local/lib/python3.6/site-packages/niworkflows/nipype/pipeline/plugins/base.py", line 101, in report_nodes_not_run
    raise RuntimeError(('Workflow did not execute cleanly. '
RuntimeError: Workflow did not execute cleanly. Check log for details
[root@ymu040-233 MRIQC_TEST]# mriqc --version
mriqc v0.9.7-rc6+7.gdbdb5cc

[logs.zip](https://github.com/poldracklab/mriqc/files/1173398/logs.zip)

**Comment by oesteban** [comment]
Hi @noelkuo,

For the error logs you attached, I would imagine that you are running mriqc with a "bare metal" installation (meaning, you are not using containers like Docker or Singularity). Is that correct?

If so, and you have full access and administration permissions to that computer, then I would strongly suggest using Docker or Singularity, since all the necessary software will be packed altogether.

If you want to keep your manual installation through pip install (or similar), then we'll need to dig into the problem. Could you please post the output of the following command?:

```Bash
antsRegistration --version
```

Basically, nipype is failing to understand the version of ANTs installed in your system. When we find what version you have, I'll post an issue in nipype to correct this problem.

**Comment by noelkuo** [comment]
Hi @oesteban 
Yes, we are running mriqc with a "bare metal" installation.
If nipype could understand the version of ANTs installed in our server, that is a great help for us,
but we will also try to install Docker.

antsRegistration --version 
ANTs Version: 2.2.0

Thanks for your assistance

**Comment by oesteban** [comment]
Just reported in nipype. While this issue is fixed, please let us know if Docker worked for you.

**Comment by noelkuo** [comment]
@oesteban 
Thanks. My advisor suggests us follow your suggestion to install Docker.
We will reinstall our system (CentOS 6 ---> CentOS 7) and try to use Docker. 
Thanks for your suggestion.

**Comment by noelkuo** [comment]
@oesteban 
We installed Docker successfully and tried to run mriqc.  However, we had a warning like #609.
[root@ymu040-233 ~]# docker -v
Docker version 17.06.0-ce, build 02c1d87
[root@ymu040-233 ~]# docker run -it poldracklab/mriqc:latest --version
mriqc v0.9.7
[root@ymu040-233 ~]# docker run -it --rm -v /root/Desktop/data/ds002:/data:ro -v /root/Desktop/output:/output poldracklab/mriqc:latest /data /output participant --participant_label 01 --no-sub

2017-07-29 08:42:57,691 interface:WARNING AFNI is outdated, detected version Debian-16.2.07~dfsg.1-5~nd16.04+1 and AFNI_17.2.05 is available.
......
2017-07-29 08:50:56,629 mriqc.report:WARNING Building bold report: no exclude index was found

In addition, we can get every image(T1w, T2w, bold) .html In the reports file, but we can't view it.
It showed "The file at /root/Desktop/data/reports/sub-01_T1w.html is not readable"

Any suggestion on this? Which step did we lose?

**Comment by oesteban** [comment]
Hi @noelkuo, is this still an issue?
